_ft/ersatz-fixtures.yml

```diff
@@ -27,12 +27,28 @@ fixtures:
         annotations:
```

Comment:
Should we add fixture for the new endpoint?

Reply:
No, the idea of the fixtures is to provide endpoints for the dependencies of our service.
That's why I've added new requests for getting from public annotations API and the PUT request for the content UUID from the Swagger file.